Header header
float64[3] position
float64[3] velocity
float64[3] attitude
float64[3] angular_velocity
float64[3] external_forces
float64[3] external_moments
float64[3] forces_offset
float64[3] moments_offset